src/pages/species-distribution/styles.module.scss

```diff
@@ -51,6 +52,7 @@ $species-list-width: 330px;
   display: flex;
   border-bottom: 1px solid $color-black;
   padding-right: 24px;
+  overflow-y: hidden;
```

Comment:
Hey, could you explain what was the purpose of adding `overflow-y: hidden;` here? It causes a weird bug in species tabs: when switching between `Species distribution` and `Map distribution`, the height of the tabs bar changes because of this:
![ilhsa-m8lzy](https://user-images.githubusercontent.com/15097138/69624341-6f52c900-103c-11ea-9ec6-7da8b0ed3bd8.gif)

Reply:
Because of a height bug in Safari, it was either that (height changes by 1 or 2px), or you couldn’t see the borders on safari. This overflow disallows the child background to overlap the parent border.

Reply:
I checked on safari and the real problem is not with `.navBar` element but with `.distribution` element (second child of the parent). We set  the height of the `.distribution` element to 100%, but the parent also contains `.navBar` element.  That's why the `.distribution` element's height should be `100% - 70px`
![image](https://user-images.githubusercontent.com/15097138/69632624-1a688000-1047-11ea-8dcb-f2e66fb7e513.png)

So as a solution please remove this `overflow-y: hidden;` and modify `distribution/styles.module.scss` file by removing `height: 100%` and adding `flex: 1` to the `.distribution` class:
![image](https://user-images.githubusercontent.com/15097138/69632124-91e9df80-1046-11ea-8ef0-b7b5fdc0b1dc.png)

This should fix the margin issue on safari without changing the height of the navbar, let me know how it goes :wink:

Reply:
So just to clarify I'm not missing anything: 
1. I removed the `overflow-y: hidden` from `src/pages/species-distribution/styles.module.scss`
2. I removed `height: 100%;` to `.distribution` on `src/pages/distribution/styles.module.scss`
3. I added `flex: 1;` to the same class, same file.

The issue is still there, maybe we can take a look at it together.
<img width="759" alt="image" src="https://user-images.githubusercontent.com/7013170/69634511-a3d17f80-1052-11ea-8dfa-3bb035e06393.png">

Reply:
Moving `border-bottom` from `.navBar` to children fixed the issue, my fault :pray: sorry